.github/workflows/sync_branches.yml

```diff
@@ -0,0 +1,31 @@
+name: Sync Branches
+on:
+  push:
+    branches:
+      - "2.4"
+      - "2.5"
```

Comment:
Is is possible to use a pattern here, like you have in the labeller or does the API expect a static branch name?

Reply:
I think it's fnmatch syntax (glob-like), so we could use `[0123456789].[0123456789]`. But it's a bit pointless since we still need to hardcode the from/to branch mapping below.